api/organisations/admin.py

```diff
@@ -13,30 +17,37 @@
 from projects.models import Project
 
 
-class ProjectInline(admin.StackedInline):  # type: ignore[type-arg]
+class OrganisationWithAnnotations(Protocol):
+    num_users: int
+    num_projects: int
```

Comment:
Is there a way to also inherit the rest of the attributes from the Organisation model?

Reply:
Can we also move this into `organisations/types.py`?